src/components/CompareResults/TableHeader.tsx

```diff
@@ -311,21 +310,50 @@ function TableHeader({
   };
 
   function renderColumnHeader(header: CompareResultsTableColumn) {
-    if ('sortFunction' in header && 'filter' in header) {
-      return (
-        <ButtonGroup
-          variant='contained'
-          color='tableHeaderButton'
-          disableElevation
-        >
+    const content = (() => {
```

Comment:
please create a separate named function, it's much more readable :) thanks

Reply:
alternatively, use a variable:
```js
let content = header.name;
if (...) {
  content = ...
} else if (...) {
  content = ...
} else...

if (header.tooltip) {
  return <Tooltip>...</Tooltip>
}
return content;
```

Reply:
It's important to remember that "simple" (and "readable") is more important than "clever".